src/nvim/main.c

```diff
@@ -207,6 +207,30 @@ void early_init(mparm_T *paramp)
   init_signs();
 }
 
+#ifdef MSWIN
+/// Set Windows console icon
+void SetWin32ConsoleIcon()
+{
+  HWND hWnd = GetConsoleWindow();
```

Comment:
`NULL` check?

Reply:
I moved some lines around. I "think" it's okay, null-checked now.